code/modules/organs/organ.dm

```diff
@@ -290,9 +290,13 @@ var/list/organ_cache = list()
 	loc = get_turf(owner)
 	processing_objects |= src
 	rejecting = null
-	var/datum/reagent/blood/organ_blood = locate(/datum/reagent/blood) in reagents.reagent_list
-	if(!organ_blood || !organ_blood.data["blood_DNA"])
-		owner.vessel.trans_to(src, 5, 1, 1)
+	if (!reagents)
```

Comment:
nope.

The first check creates it if it's not there, thats what stops the runtime error.

The second check, which is really absurdly paranoid, just checks to ensure that the creation didnt fail somehow.

Because limb removal is an infrequent and usually player-initiated action, performance isnt important here so paranoid safety is the way to go